.eslintrc.json

```diff
@@ -14,8 +14,9 @@
         }
       },
     "rules": {
-        "react/jsx-filename-extension": [1, {"extensions": [".js"]}],
+        "react/jsx-filename-extension": [1, {"extensions": [".js","jsx"]}],
         "react/prefer-stateless-function": "off",
+        "react/no-array-index-key": "off",
```

Comment:
Why was this rule added?

Reply:
I was supposed to remove it. Sorry about that